lib/registry.ts

```diff
@@ -61,19 +67,14 @@ export function unregisterAppTranslations(appId: string) {
  */
 export function getAppTranslations(appId: string): AppTranslations {
 	if (
-		typeof window._oc_l10n_registry_translations === 'undefined'
-		|| typeof window._oc_l10n_registry_plural_functions === 'undefined'
+		typeof window._oc_l10n_registry_translations?.[appId] === 'undefined'
+		|| typeof window._oc_l10n_registry_plural_functions?.[appId] === 'undefined'
 	) {
-		console.warn('No OC L10N registry found')
-		return {
-			translations: {},
-			pluralFunction: (number: number) => number,
-		}
+		console.warn(`No translation for appId "${appId}" have been registered`)
 	}
-
 	return {
-		translations: window._oc_l10n_registry_translations[appId] || {},
-		pluralFunction: window._oc_l10n_registry_plural_functions[appId],
+		translations: window._oc_l10n_registry_translations?.[appId] ?? {},
+		pluralFunction: window._oc_l10n_registry_plural_functions?.[appId] ?? ((number: number) => number),
 	}
 }
```

Comment:
Update warns logic here.

Since it doesn't matter if "no app translation has been registered" or "only `appId` hasn't been registered", check both situations with optional chaining.

Warn is always about concrete `appId`, as there is no "general translation registration." It is always some app translation registration.

Also, return default stub for both `translations` and `pluralFunction` if there is no value.